commit 18388923a155529c4786ae82f359abecbc5d0e0c
Author: Chris Burroughs <chris.burroughs@joyent.com>
Date:   2019-07-31T10:37:27-04:00 (10 weeks ago)
    
    TRITON-1805 volapi over enthusastic about compiling sdcnode
    Reviewed by: Todd Whiteman <todd.whiteman@joyent.com>
    Approved by: Todd Whiteman <todd.whiteman@joyent.com>

diff --git a/Makefile b/Makefile
index bd7b531..4341345 100644
--- a/Makefile
+++ b/Makefile
@@ -42,7 +42,8 @@ ifeq ($(shell uname -s),SunOS)
 	include ./deps/eng/tools/mk/Makefile.node_prebuilt.defs
 	include ./deps/eng/tools/mk/Makefile.agent_prebuilt.defs
 else
-	include ./deps/eng/tools/mk/Makefile.node.defs
+	NPM_EXEC :=
+	NPM = npm
 endif
 include ./deps/eng/tools/mk/Makefile.smf.defs
 
